Justfile

```diff
@@ -42,11 +42,14 @@ integration-tests:
 	NATS_URL=$(NATS_URL) cargo test --workspace
 
 deps:
-	npm install -g apex-template prettier ts-node
+	npm install -g apex-template prettier ts-node commitlint conventional-changelog-conventionalcommits
```

Comment:
Please add to the PR description that you added these deps

Reply:
Done